ext/odbc/php_odbc.c

```diff
@@ -1594,12 +1578,12 @@ PHP_FUNCTION(odbc_fetch_row)
 	if (rc != SQL_SUCCESS && rc != SQL_SUCCESS_WITH_INFO) {
 		RETURN_FALSE;
 	}
-
-	if (!pv_row_is_null) {
+#ifdef HAVE_SQL_EXTENDED_FETCH
```

Comment:
this looks like an omission. The result of the affected functions already used the same `#ifdef`s